package.json

```diff
@@ -10,20 +10,8 @@
     "url": "https://github.com/paularmstrong/normalizr.git",
     "type": "git"
   },
-  "keywords": [
-    "flux",
-    "redux",
-    "normalize",
-    "denormalize",
-    "api",
-    "json"
-  ],
-  "files": [
-    "dist/",
-    "index.d.ts",
-    "LICENSE",
-    "README.md"
-  ],
+  "keywords": ["flux", "redux", "normalize", "denormalize", "api", "json"],
```

Comment:
those changes are caused by some auto-formatting - im not sure, but I think my yarn does it automatically when installing new deps (which I did for a brief moment, but removed later)

I can revert this file if you want

Reply:
I'm in the process of adding prettier auto-formatting on pre-commit :) Don't worry about it.